app/routes/planet_routes.py

```diff
@@ -0,0 +1,62 @@
+from flask import Blueprint, abort, make_response, request
+from app.models.planet import planets
+# from ..db import db
+
+planets_bp = Blueprint("planets_bp", __name__, url_prefix="/planets")
+
+@planets_bp.get("")
+def get_all_planets():
+
+    planets_response = []
+    for planet in planets:
+        planets_response.append(
+            {
+                "id": planet.id,
+                "name": planet.name,
+                "description": planet.description
+            }
+        )
+    return planets_response
+
+@planets_bp.get("/<planet_id>")
+def get_one_planet(planet_id):
+    planet = validate_planet(planet_id) 
+
+    for planet in planets:
+        if planet.id == int(planet_id):
+            return {
+                "id": planet.id,
+                "name": planet.name,
+                "description": planet.description,
+            }
```

Comment:
After your model has been validated, we don't need to loop through all the planets again to see if the id matches. Have a look at the `validate_planet` function and see that you already check to see if the id matches on line 40-42. 

```suggestion
            return {
                "id": planet.id,
                "name": planet.name,
                "description": planet.description,
            }
```